recipes/lib3mf/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -x
+
+mkdir build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DCMAKE_INSTALL_LIBDIR="${PREFIX}/lib" \
```

Comment:
Use CMAKE_ARGS for the most standard CMake arguments on conda-forge.
Also, please use zlib & co from conda-forge.
```suggestion
    ${CMAKE_ARGS} \
    -DUSE_INCLUDED_ZLIB=OFF -DUSE_INCLUDED_LIBZIP=OFF -DUSE_INCLUDED_SSL=OFF \
```

Reply:
Should I use %CMAKE_ARGS% in bld.bat?

Reply:
No, we don't have that for Windows yet.